### DISCUSSION TITLE
Welcome to alfred3 discussions

### DISCUSSION THREAD

**Discussion by jobrachem** [ctreffe/alfred#81] [General]
## 👋 Welcome

  We’re using Discussions as a place to connect with other alfred3 experimenters. We hope that you:

  * Ask questions you’re wondering about.
  * Help people out, if you know the answer to their problem.
  * Share your ideas and projects.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Adsecularis** [comment]
Hello 👋

My name is Jaron or Adsecularis on GitHub. Adsecularis, also known as Tech-Thralls, are menial laborers used by the Adeptus Mechanicus from Warhammer 40k.

I am a psychology student at the university of Hildesheim and work as a research assistant for the Social-, Organizational- and Economic Psychology Workgroup. As a research assistant I saw a bit of a correlation between me and Adsecularis, hence the name 😁 _please don't fire me_

I made this account to get help from and give help to others in coding experiments with alfred3. Since I found this Discussion a while after I had asked questions and given answers I can say from experience that both worked out well. 👍

**Reply by ctreffe** [reply]
Hey @Adsecularis ,

we are very happy to have you on board with us.

_"Damned be the man who reveres not the Omnissiah, your ignorance shall be your doom!"_
